packages/core/documentation/package.json

```diff
@@ -60,17 +70,7 @@
     "extends": "../../../package.json"
   },
   "distSizeChecker": {
-    "errorLimit": "500KB",
-    "warningLimit": "450KB"
-  },
-  "peerDependencies": {
-    "@kong-ui-public/i18n": "workspace:^",
-    "@kong/kongponents": "9.0.0-alpha.113",
-    "vue": "^3.4.21"
-  },
-  "dependencies": {
-    "@kong-ui-public/document-viewer": "workspace:^",
-    "@kong-ui-public/entities-shared": "workspace:^",
-    "@kong/icons": "^1.8.14"
+    "errorLimit": "30MB",
+    "warningLimit": "25MB"
```

Comment:
This is expected, the new `@kong/markdown` package is essentially adding `mermaid` support as well as enabling many more languages by default. We can shave this down in the future once we determine the langs we want to support out of the box.